integration-test-suite/test-grid-job/deployment.properties

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+ApimHostname=ob-am
+IamHostname=ob-iam
+BiHostname=ob-bi
+BrowserAutomation.HeadlessEnabled=true
+ApiVersion=1.3.0
+DbType=mysql
+DBServerHost=localhost
+DBUsername=root
+DBPassword=p@ssw0rd
+DBDriverClass=com.mysql.jdbc.Driver
+#Following configuration is required only for Oracle DB
+OracleDBSID=xe
+#Value Should be mac or linux
+OSName=mac
+ApimServerPath=/Users/malshanisenarathne/Documents/CDS_Toolkit/MigrationNew/OB2_AM/wso2am-4.2.0
```

Comment:
Shall we change this as a config with a placeholder?

Reply:
Check other places as well

Reply:
Fixed.